purchases/src/main/kotlin/com/revenuecat/purchases/VirtualCurrencyInfo.kt

```diff
@@ -0,0 +1,34 @@
+package com.revenuecat.purchases
+
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
+import org.json.JSONObject
+
+/**
+ * A class representing information about a virtual currency in the app.
+ *
+ * @property balance The current balance of the virtual currency.
+ */
+@Parcelize
+data class VirtualCurrencyInfo internal constructor(
+    val balance: Long,
```

Comment:
I'm wondering if it would be worth adding the currency id/name here as well... It's available as the key of the map in `CustomerInfo` but might be more developer friendly to duplicate it here I think. Can be done in follow-up PRs too!